webpack.common.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     context: __dirname,
     entry: {
         main: './assets/js/app.js',
+        head_async: ['lazysizes'],
```

Comment:
Loading lazysizes module in the head script. This file should only be used for mission-critical head JS.